woocommerce/woocommerce
# [Experimental] Utilize WP generated class names to style interactivity stock filter block.

### Changes proposed in this Pull Request:

@dinhtungdu and I agreed that we don't need to create our own unique class names or use bem style syntax, its overly verbose and doesn't add anything.

Instead we scope all styles to the class name generated for the block by WP, and also remove the BEM style prefix from any scoped class names. (To keep PR's small and testable I'm addressing one block at a time.)

Additionally, I noticed a lot of copy pasted styles from the old block that we don't use or need. 

This is part of https://github.com/woocommerce/woocommerce/issues/43167

### How to test the changes in this Pull Request:

1. Much like in https://github.com/woocommerce/woocommerce/pull/44069, edit the product catalog or a page and alongside product collection add stock status filter.
2. Also in editor try different modes like dropdown/multiple etc
3. Check that styling looks good in editor and frontend.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 
[Experimental] Unify class naming in new interactivity stock filter block.


#### Comment 

</details>
